docs/output.md

```diff
@@ -2,42 +2,228 @@
+      - `INDEL_ALL_variants_by_tool_happy.png`: Summary plot for callers on TP, FP and FN numbers of all INDELs in happy
+      - `INDEL_PASS_metric_by_tool_happy.png`: Summary plot for callers on precision, recall and F1 of only PASSed INDELs in happy
+      - `INDEL_PASS_variants_by_tool_happy.png`: Summary plot for callers on TP, FP and FN numbers of only PASSed INDELs in happy
+      - `SNP_ALL_metric_by_tool_happy.png`: Summary plot for callers on precision, recall and F1 of all SNPs in happy
+      - `SNP_ALL_variants_by_tool_happy.png`: Summary plot for callers on TP, FP and FN numbers of all SNPs in happy
+      - `SNP_PASS_metric_by_tool_happy.png`: Summary plot for callers on precision, recall and F1 of only PASSed SNPs in happy
+      - `SNP_PASS_variants_by_tool_happy.png`: Summary plot for callers on TP, FP and FN numbers of only PASSed SNPs in happy
+    - `rtgtools/`
+      - `metric_by_tool_rtgtools.png`: Summary plot for callers on precision, recall and F1 in rtgtools
+      - `variants_by_tool_rtgtools.png`: Summary plot for callers on TP, FP and FN numbers in rtgtools
+  - `indel/`
+    - `sompy/`
+      - `metric_by_tool_sompy.png`: Summary plot for callers on precision, recall and F1 of indels in sompy
+      - `variants_by_tool_sompy.png`: Summary plot for callers on TP, FP and FN numbers of indels in sompy
+  - `snv/`
+    - `sompy/`
+      - `metric_by_tool_sompy.png`: Summary plot for callers on precision, recall and F1 of SNVs in sompy
+      - `variants_by_tool_sompy.png`: Summary plot for callers on TP, FP and FN numbers of SNVs in sompy
+- `tables/`
+  - `cnv/`
+    - `wittyer.cnv.summary.csv`: Summary of performance stats from callers
+  - `sv/`
+    - `truvari.sv.summary.csv`: Summary of performance stats from callers
+    - `svbenchmark.sv.summary.csv`: Summary of performance stats from callers
+  - `small/`
+    - `happy.sv.summary.csv`: Summary of performance stats from callers
+    - `rtgtools.sv.summary.csv`: Summary of performance stats from callers
+  - `indel/`
+    - `sompy.indel.summary.csv`: Summary of performance stats from callers
+    - `sompy.indel.regions.csv`: Summary of performance stats split by region bins from callers
+  - `snv/`
+    - `sompy.snv.summary.csv`: Summary of performance stats from callers
+    - `sompy.snv.regions.csv`: Summary of performance stats split by region bins from callers
+- ## `datavzrd/`
+
+</details>
+<summary>Output files</summary>
+
+- benchmark tool
+  - static
+  - test
+  - index.html: HTML file to open in the browser for datavzrd spedific tables and visualizations.
+
+### References
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `references/`
+  - `dictionary`
+    - `*.dict`: Dictionary file is the output of PICARD CREATESEQUENCEDICTIONARY. This file can be saved and reused further.
+  - `sdf`
+    - `*.sdf`: Sdf file is the output of RTGTOOLS FORMAT. This file can be saved and reused further.
+
+</details>
```

Comment:
Not sure what the structure that you meant was here, but this looks a bit weird: https://nf-co.re/variantbenchmarking/dev/docs/output/#summary-statistics

Reply:
fixed it. :)